x-pack/plugins/infra/public/components/logging/log_time_controls.tsx

```diff
@@ -88,6 +89,7 @@ class LogTimeControlsUI extends React.PureComponent<LogTimeControlsProps> {
 
   private startLiveStreaming = () => {
     this.props.startLiveStreaming(5000);
+    this.props.setSurroundingLogsId(null);
```

Comment:
Similar to my thoughts about the `jumpToTarget` prop above, how about we just modify the passed in `startLiveStreaming` prop to also reset the id? Then the time picker wouldn't have to know about the entanglement between live streaming and the surrounding logs id.